content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md

```diff
@@ -0,0 +1,150 @@
+
+![](/img/ws-image4.png)
+
+![](/img/ws-image5.png)
+
+The **hpe\_ccs\_attribute** will always start with **version_1#**. First, we’ll configure the attributes for GreenLake CCS, then Central. First enter the PCID for the account, followed by the GreenLake application ID. This will always be **00000000-0000-0000-0000-000000000000**. Then followed by the role name and **ALL_SCOPES**. Next will be the Aruba Central info. Start with the **app cid**, then the role name (IE Aruba Central Administrator) then **ALL_SCOPES**.
+
+Example:
+
+**version_1#5b0ec0e8c4f422eca232ba72799953ac:00000000-0000-0000-0000-000000000000:Account Administrator:ALL_SCOPES:683da368-66cb-4ee7-90a9-ec1964768092:**
+
+**Aruba Central Administrator:ALL_SCOPES**
+
+If you want to add additional GreenLake applications or if you have multiple Aruba Central accounts, you can add them as well. Just follow the same syntax as before. Once you have the attribute defined, enter it into the SAML attribute statement in Okta as shown below.
+
+![](/img/ws-image6.png)
+
+2﻿. Complete the setup.
+
+![](/img/ws-image7.png)
+
+Click Next and Select “Internal App” then Finish
+
+**Step 3:** **Export the SAML 2.0 IdP metadata**
+
+1. Click Next – Configure the Sign On settings
+
+   Two options available: **View Setup Instructions** which steps through the SAML config or **Identity Provider metadata** will produce an XML that can be loaded into Aruba Central.
+
+   Suggestion: Click **Identity Provider metadata** and save the XML data to a file.
+
+   ![](/img/ws-image9.png)
+2. C﻿lick Next
+3. Select Internal app and Click Finish
+
+**Step 4: Create SAML Authorization Profile in GreenLake Cloud Platform**
+
+1. Log into GreenLake and click Menu > Manage > Authentication and Click Set Up SAML Connection.
+
+   *Before you can add a new SAML configuration, you must have at least one user account with that domain already enabled in GreenLake. Also, you must be logged into GreenLake with an account from that domain in order to enable SSO for that domain.*
+
+   ![](/img/ws-image10.png)
+2. Type in the domain you want to enable SSO on:
+
+   ![](/img/ws-image11.png)
+3. Input the metadata from the above step
+
+   While GreenLake does support manually entering this info, it’s recommended to simply upload the XML metadata that was downloaded in the previous step. Select Metadata File and select the XML. Then click Next.
+
+   ![](/img/ws-image12.png)
+4. Enter the SAML attributes to match what was entered in Okta and set the idle timeout value as well.
+
+   ![](/img/ws-image13.png)
+5. Then click Next.
+6. Create a recovery user in the event SSO fails, an admin will still be able to access GreenLake.
+
+   ![](/img/ws-image14.png)
+
+   Congratulations SSO will now be enabled for GreenLake as well as the Aruba Central application. Log out and on the GreenLake home page, click Sign in with SSO.
+
+**Testing and Troubleshooting:**
+
+On the GreenLake Cloud Platform home page, Click Sign in with SSO.
```

Comment:
Add "HPE" proir to "GreenLake". Change "Click" to "click". Change "in" to "In", since that's what's shown in the image. So it should read:
"On the HPE GreenLake Cloud Platform home page, click Sign In with SSO."

Reply:
Updated:
On the HPE GreenLake Cloud Platform home page, click Sign In with SSO.